docs/SUMMARY.md

```diff
@@ -15,7 +15,7 @@
 ## User Guide
 
 * [Creating Application](user-guide/creating-application/README.md)
-  * [Git Material](user-guide/creating-application/git-material.md)
+  * [Git Repository](user-guide/creating-application/git-repository.md)
```

Comment:
Do not rename the files as these are linked from within the product and can affect SEO too as these links are already crawled on google. Leave the filenames as they are unless we are able to find a way for implementing 301 redirects